src/app/app-routing.module.ts

```diff
@@ -1,9 +1,9 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { LoginComponent } from './login/login.component'
-import { CallbackComponent } from './callback/callback.component'
-import { PlaylistComponent } from './playlist/playlist.component'
-import { SettingsComponent } from './settings/settings.component'
+import { LoginComponent } from './components/login/login.component'
```

Comment:
I noticed, when starting this project, that angular doesn't handle files the same as Ionic did. I don't know if this was by choice of the angular developers but I'm on board with filing components. 👍

Reply:
So if I'm understanding you correctly, you're on board for sorting components, but not for temporarily deleting the test files. If so I can revert back to my first commit then merge to master.

Reply:
yessir